docs/INSTALL-COMPLETE-GUIDE.md

```diff
@@ -21,94 +33,114 @@ Alternatively you can use an external sound card, but sometimes that doesn't see
+1. Enable SSH by adding a simple file
+    ```
+    $ touch ssh
+    ```
+1. Set up your Wifi connection
+    * Mac
+        ```
+        $ nano wpa_supplicant.conf
+        ```
+    * Windows
+        ```
+        D:\> notepad wpa_supplicant.conf
+        ```
+1. Insert the following content, update your country, Wifi credentials and save the file.
+    ```
+    country=DE
+    ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
+    update_config=1
+
+    network={
+        ssid="network-name"
+        psk="network-password"
+    }
+    ```
+1. Eject your SD card and insert it into your Raspberry Pi
 1. Start your Raspberry Pi by attaching a power supply
-1. Walk through the "Welcome to Raspberry Pi" wizard (the content of this wizard might have changed after this tutorial was created!)
-    1. Set your locale
-    1. Change your password
-    1. Proceed with with screen settings
-    1. Select your Wifi Network (You are being asked in the Phoniebox install routine whether you want to set up your Wifi again, we will skip this step then)
-    1. Software Update (optional, recommended, takes a while)
-    1. Reboot
-1. Let's enable a few more settings
-    1. Launch `Raspberry Pi Configuration` from the `Preferences` menu
-    1. A window opens with the `System` tab selected
-    1. Select `To CLI` for `Boot` option
-    1. Select `Login as user 'pi'` for `Auto login` option
-    1. Select `Wait for network` for `Network at Boot` option (optional, required for Spotify+ version)
-    1. Navigate to the `Interfaces` tab
-    1. Select `Enabled` next to `SSH`
-    1. Click `OK`
-1. Set up a static IP (optional, recommended)
-    1. Right click on the Wifi sympbol in the upper right corner of your application bar and choose `Wifi & Wired Network Settings`
-    1. Configure `interface` and `wlan0`
-    1. Check `Disabled IPv6` unless you want to provide a static IPv6 address
-    1. Fill out `IPv4` and `Router` (Gateway) options (keep `DNS Servers` and `DNS Search` empty)
-    1. Click `Apply` and `Close`
-1. Optional: If you like, you can **turn off Bluetooth** to reduce energy consumption (unless you want to use any Bluetooth devices with your Phoniebox)
-1. Shutdown your Raspberry Pi (`Application > Logout > Shutdown`)
-
-## Install Phoniebox software
+1. Login into your Raspberry Pi, username is `pi` and password is `raspberry`. If `raspberrypi.local` does not work, find out your Raspberry Pi's IP address from your router.
+    ```
+    $ ssh pi@raspberrypi.local
+    ```
+1. Update the Pi's software. This may take a bit
+    ```
+    $ sudo apt update && sudo apt full-upgrade
+    ```
+1. Reboot with `sudo reboot`
+1. Login again with SSH and open the Raspberry Pi config
+    ```
+    $ sudo raspi-config
+    ```
+1. Update the following settings
+    ```
+    1 System Options
+        S5 Boot / Auto Login -> B2 Console Autologin
+        S6 Network at Boot -> Yes
```

Comment:
This is necessary here, to complete the installation. Should you mention at the end that this can also be switched off - some are very concerned about WiFi near infants and kids.